Review.java

```diff
@@ -165,4 +165,31 @@ public static String randomAdjective()
       return randomNegativeAdj();
     }
   }
+  public static int starRating(String fileName)
+   {
+     double sentiment = totalSentiment(fileName);
+     int rating;
+     
+     if (sentiment < -3)
+     {
+       rating = 1;

```

Comment:
You can return an int without creating a variable but otherwise looks good to merge!

Reply:
Thanks for the feedback!